circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
```

Comment:
In https://github.com/Automattic/wp-calypso/commit/ac95127d5bc62f06e9dde597eb5d7657b559badc

One odd bit: passing $(JS_FILES) through made the linter bomb. Unsure why.